### PR TITLE
Replace ArrowLeft etc. with an ascii arrow

### PR DESCRIPTION
**Pull request by philrenaud**
image

Replaces "ArrowUp" with `↑` in the modal, and so forth.

### CODE REVIEW

File: ui/app/helpers/clean-keycommand.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  ArrowDown: '↓',
+  '+': ' + ',
+  // Enter: '⏎',
+  // Tab: '⇥',
+  // Space: '␣',
+  // Shift: '⇧',
+};
+
+export default helper(function cleanKeycommand([part] /*, named*/) {
+  console.log('cleaning', part);
```
Comment (ChaiWithJai) [inline ui/app/helpers/clean-keycommand.js:17]: `blocking`:  remove the `log` statement.

File: ui/app/helpers/clean-keycommand.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  Object.keys(KEY_ALIAS_MAP).forEach((key) => {
+    part = part.replace(key, KEY_ALIAS_MAP[key]);
```
Comment (ChaiWithJai) [inline ui/app/helpers/clean-keycommand.js:19]: `suggestion` `non-blocking`:  Generally I'm not a fan of mutating an argument passed a function.

File: ui/app/helpers/clean-keycommand.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  ArrowUp: '↑',
+  ArrowDown: '↓',
+  '+': ' + ',
+  // Enter: '⏎',
+  // Tab: '⇥',
+  // Space: '␣',
+  // Shift: '⇧',
+};
+
+export default helper(function cleanKeycommand([part] /*, named*/) {
```
Comment (ChaiWithJai) [inline ui/app/helpers/clean-keycommand.js:16]: `question`:  Why do we pass in a `key` in the template, but we name the argument `part`?

File: ui/app/helpers/clean-keycommand.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  console.log('cleaning', part);
+  Object.keys(KEY_ALIAS_MAP).forEach((key) => {
+    part = part.replace(key, KEY_ALIAS_MAP[key]);
+  });
+
+  return [part];
```
Comment (ChaiWithJai) [inline ui/app/helpers/clean-keycommand.js:22]: `question`:  This returns a list that will be rendered inside of a `span`. If we're returning a list, shouldn't invoke this template helper and assign the return value to a variable and then `each` over that?

**Review by ChaiWithJai — APPROVED** [review]

File: ui/app/helpers/clean-keycommand.js (hunk 3 above)
Comment (philrenaud) [reply ui/app/helpers/clean-keycommand.js:16]: The key might be a sequence, like `["g", "v"]` for `go to variables`. `g` and `v` would be individual `part`s.

Comment (philrenaud) [reply ui/app/helpers/clean-keycommand.js:16]: Hm you know what? You bring up a good point here. I'm using too many words to mean roughly the same thing. Will bring them all down to `part`.

File: ui/app/helpers/clean-keycommand.js (hunk 4 above)
Comment (ChaiWithJai) [reply ui/app/helpers/clean-keycommand.js:22]: Wow! So checkout how fail-safe Handlebars is.

image

**Pull request merged**